docs/feature_rgblight.md

```diff
@@ -352,6 +352,7 @@ rgblight_sethsv(HSV_GREEN, 2); // led 2
 |`rgblight_step_noeeprom()`                  |Change the mode to the next RGB animation in the list of enabled RGB animations (not written to EEPROM) |
 |`rgblight_step_reverse()`                   |Change the mode to the previous RGB animation in the list of enabled RGB animations |
 |`rgblight_step_reverse_noeeprom()`          |Change the mode to the previous RGB animation in the list of enabled RGB animations (not written to EEPROM) |
+|`rgblight_reset_from_eeprom()`              |Reload the effect configuration (enabled, mode and color) from EEPROM |
```

Comment:
`rgblight_reload()` might be clearer; "reset" to me implies going back to default settings.